src/sentry/models/exporteddata.py

```diff
@@ -57,18 +57,20 @@ def status(self):
         else:
             return ExportStatus.Valid
 
-    @property
-    def date_expired_string(self):
-        if self.date_expired is None:
-            return None
-        return self.date_expired.strftime("%-I:%M %p on %B %d, %Y (%Z)")
-
     @property
     def payload(self):
         payload = self.query_info.copy()
         payload["export_type"] = ExportQueryType.as_str(self.query_type)
         return payload
 
+    def get_date_string(self, date_field):
+        if not hasattr(self, date_field):
+            raise ValueError(u"Unrecognized field {} on ExportedData object".format(date_field))
+        if getattr(self, date_field) is None:
+            return None
+        else:
+            return getattr(self, date_field).strftime("%-I:%M %p on %B %d, %Y (%Z)")
```

Comment:
Generalized the property I'd previously added so that it could be used in both email functions with minimal code duplication

Reply:
@leeandher Can you add a comment for an example of what this string looks like? Hard to see from the code alone.

Reply:
Done!